### DISCUSSION TITLE
Autolink references need more delimiters

### DISCUSSION THREAD

**Discussion by cknowles-moz** [community/community#40371] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Our environment, which internally uses bugzilla, we'd like the autolink references to bugzilla bugs from GitHub to work the same way that bugzilla bugs can autolink. Specifically - using "bug ####" to reference the bug. https://bugzilla.readthedocs.io/en/latest/using/tips.html#autolinkification

Note that we have gotten the autolink working with hyphen as the delimiter - but from long practice and habit other systems we have use " " as the delimiter - so it'd be great if GitHub did too.

**Comment by ossanna16** [comment]
Hi there @cknowles-moz and welcome to our community! Thank you for the feedback!
